demo-app/src/main/java/io/opentelemetry/android/demo/OtelDemoApplication.kt

```diff
@@ -40,24 +37,8 @@ class OtelDemoApplication : Application() {
                 .setDiskBufferingConfig(diskBufferingConfig)
 
         // 10.0.2.2 is apparently a special binding to the host running the emulator
-        val spansIngestUrl = "http://10.0.2.2:4318/v1/traces"
-        val logsIngestUrl = "http://10.0.2.2:4318/v1/logs"
-        val otelRumBuilder: OpenTelemetryRumBuilder =
-            OpenTelemetryRum.builder(this, config)
-                .addSpanExporterCustomizer {
-                    OtlpHttpSpanExporter.builder()
-                        .setEndpoint(spansIngestUrl)
-                        .build()
-                }
-                .addLogRecordExporterCustomizer {
-                    OtlpHttpLogRecordExporter.builder()
-                        .setEndpoint(logsIngestUrl)
-                        .build()
-                }
-                // TODO: This should NOT be necessary if it's in the runtime classpath...
-                .addInstrumentation(SessionInstrumentation())
         try {
-            rum = otelRumBuilder.build()
+            rum = OpenTelemetryRumInitializer.initialize(this, "http://10.0.2.2:4318")
```

Comment:
`OpenTelemetryRumInitializer` is an object (singleton) but looks like it should be just a function or extension?
the single class holds no state or anything.

Reply:
You're right. The reason I went with an object is to try and provide a consistent experience for users calling either from Kotlin or Java code. Though I'm open to other approaches. If we turned it into a standalone function, for example, would it be a way for people from Java to still call it this way: `OpenTelemetryRumInitializer.initialize(...)`?

We could also decide not to think/worry about Java usage apis, though I'd like to get some feedback on whether that could cause issues or not.

Reply:
as a kotlin object, i think Java users have to do eg `OpenTelemetryRumInitializer.INSTANCE.initialize`
I think for `OpenTelemetryRumInitializer.initialize` you'd need a `@JvmStatic` annotation within a companion object

Reply:
yeah i was not sure about being within the companion object anyway, if it works, all good